tests/unit/Entities/AudioTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * This file is part of the TelegramBot package.
+ *
+ * (c) Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Longman\TelegramBot\Tests\Unit;
+
+use Longman\TelegramBot\Entities\Audio;
+use Longman\TelegramBot\Exception\TelegramException;
+use Longman\TelegramBot\Tests\Unit\TestHelpers;
+
+/**
+ * @package         TelegramTest
+ * @author          Baev Nikolay <gametester3d@gmail.com>
+ * @copyright       Avtandil Kikabidze <akalongman@gmail.com>
+ * @license         http://opensource.org/licenses/mit-license.php  The MIT License (MIT)
+ * @link            http://www.github.com/akalongman/php-telegram-bot
+ */
+class AudioTest extends TestCase
+{
+    /**
+    * @var array
+    */
+    private $record;
+    /**
+    * Set Up
+    */
+    public function setUp()
+    {
+        $this->record = TestHelpers::getFakeRecordedAudio();
+    }
+    /**
+    * Testing base stage with data object creating
+    */
+
+    public function testBaseStageAudio()
+    {
+        $audio = new Audio($this->record);
+        $this->assertInstanceOf('Longman\TelegramBot\Entities\Audio', $audio);
+    }
+    
+    /**
+    * Test base stage without duration property
+    *
+    * @expectedException Longman\TelegramBot\Exception\TelegramException
+    */
+    public function testBaseStageWithoutDuration()
+    {
+        $this->record['duration'] = null;
+        new Audio($this->record);
+    }
+
+   /**
+    * Test base stage without duration property
+    *
+    * @expectedException Longman\TelegramBot\Exception\TelegramException
+    */
+    public function testBaseStageWithoutFileId()
```

Comment:
Same as `testBaseStageWithoutDuration()`, needs to be fixed.
Also docblock needs updating.